README.md

```diff
@@ -5,7 +5,23 @@
 We use [Sass](https://sass-lang.com/) as css preprocessor.
 
 #### Selectors
-Only use class names for styling, no id's.
+- Only use class names for styling, no id's
+- Use snake case for selectors (eg `.image-wrapper`)
+- When targeting any element add a class to it when possible to be more specific (eg `.list` instead of `ul`) to retain styles when the html element is changed (eg from `ul` to `ol`)
+- When using mixins always put them as the first properties of the selector, it looks neat and this way they can be (partially) overridden when needed:
```

Comment:
I think this is forced by style-lint, so doesn't have to be included here.

Reply:
Not in vue-skeleton right now, not sure about Muban. We could add it if there are no arguments against it.